cekit/cli.py

```diff
@@ -141,6 +141,11 @@ def parse(self):
                             default="image.yaml",
                             help="path to image descriptor file, default: image.yaml")
 
+        parser.add_argument('--add-help',
```

Comment:
please make it also key in ``~/.cekit/config`` so it  can be set for whole tool.

Reply:
np (added to my TODO checklist)